tests/cilium_tls_tcp_integration_test.cc

```diff
@@ -18,6 +18,7 @@
 #include "envoy/common/exception.h"
 #include "envoy/event/dispatcher.h"
 #include "envoy/extensions/transport_sockets/tls/v3/tls.pb.h"
+#include "envoy/http/codec.h"
```

Comment:
unused

Reply:
Have to keep this, otherwise I'll get:
```
tests/cilium_tls_tcp_integration_test.cc:127:61: error: no header providing "Envoy::Http::CodecType::HTTP1" is directly included [misc-include-cleaner,-warnings-as-errors]
   21 |       config.upstream_protocol_ = FakeHttpConnection::Type::HTTP1;
```